src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -152,6 +153,60 @@ static DataTestUtility()
             }
         }
 
+        public static string SetupFileStreamDB()
```

Comment:
Since this method is called from outside classes, passing those would mean code like this:

```cs
DataTestUtility.SetupFileStreamDB(DataTestUtility.FileStreamDirectory, DataTestUtility.TCPConnectionString)
```

I prefer accessing local properties directly.